helpers/package.json

```diff
@@ -2,7 +2,8 @@
     "name": "chart.js-helpers",
     "private": true,
     "description": "helper package",
+    "type": "module",
     "main": "helpers.js",
-    "module": "helpers.mjs",
-    "types": "helpers.mts"
-}
\ No newline at end of file
+    "exports": "./helpers.js",
```

Comment:
Should we still keep the `module` in case some bundler does not work with `exports`?

Reply:
@kurkle In this case bundler will look at `main` field, where is esm file.

Reply:
👍